src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -212,7 +212,7 @@ func RemoveNetwork(c Client) error {
 
 func CreateNode(
 	c Client,
-	nodeID, kafkaPort, proxyPort, rpcPort, metricsPort uint,
+	nodeID, kafkaPort, proxyPort, schemaRegPort, rpcPort, metricsPort uint,
```

Comment:
@dimitriscruz - can we schedule time to have the schema be in the same certs as the HTTP Proxy